src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -269,6 +275,11 @@ FormPage.propTypes = {
       uiSchema: PropTypes.object.isRequired,
       updateFormData: PropTypes.func,
     }),
+    formConfig: PropTypes.shape({
```

Comment:
I think just delete this and you're good to go (unless I'm missing how this is still used)

Reply:
This was a separate propType before, but since the refactor this propType now belongs to route, so looks like I need to define it, otherwise I get an eslint issue: `'route.formConfig' is missing in props validation eslint[react/prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/prop-types.md)`.  Looks like `PropTypes.object` could also work here